doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-The standalone editors are distributed in a self-contained library that provides an all-in-one JavaScript file for each editor. The JavaScript file uses a comprehensive API to set and control the editor. You can install the standalone editors in two ways:
+The following list provides a summary of the two new data set providers in {CENTRAl}:
 
-* Download each JavaScript file manually
-* Use the NPM package
+* You can use the *Prometheus* data set provider which generates a data set using the Prometheus query.
+* You can use the *Kafka* data set provider which generates a data set using metrics from Kafka broker, consumer, or producer.
 
-For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_] and {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
+For more information, see {URL_MANAGING_SETTINGS}#assembly-configuring-central[_{CONFIGURING_CENTRAL}_].
 
-== Support for VSCode extension
+=== Test scenarios
 
-{PRODUCT} now provides support for VSCode extension enabling you to view and design BPMN and DMN models in Visual Studio Code (VSCode). The VSCode extension in {PRODUCT} requires VSCode 1.46.0 or later.
+The test scenarios designer in {CENTRAL} is now available with the new features supported in {PRODUCT} {PRODUCT_VERSION}.
 
-You can install the {PRODUCT} VSCode extension using:
+* The test scenarios designer in {CENTRAL}  now supports the nested enumeration data type attributes in rule-based test scenarios.
 
-* VSCode user interface
+* In test scenarios designer, you can now see the actual test results instead of a generic message in a list or a map collection editor. It is now possible to monitor the following information in a collection editor:
+** The wrong value
+** The path to reach out to the wrong value
+** The correct value
 
-. Open *Extensions* menu option.
-. Search for and install *Red Hat Business Automation Bundle* extension.
+For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-test-scenarios[_{TESTING_DECISION_SERVICE}_].
 
-* Terminal
+ifdef::PAM[]
 
-. Launch VSCode Quick Open.
-. Enter the following command and press *Enter*.
-+
-`ext install redhat.vscode-extension-red-hat-business-automation-bundle`
+=== New `user` parameter in KIE REST API endpoint
 
-For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_] and {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
+You can now use the new `user` parameter in a KIE REST API endpoint to bypass the enabled authenticated user. The `user` parameter is added for adding or removing the users or groups to a task.
 
+endif::[]
 
-ifdef::PAM[]
+=== PMML trusty implementation
```

Comment:
Think this should be moved under Kie-server

Reply:
Updated